h/static/styles/partials-v2/_base.scss

```diff
@@ -52,6 +52,13 @@ $max-phone-width: 500px;
 $tablet-width: 768px;
 $touch-target-size: 44px;
 
+// Misc
+// ----
+$search-result-container-margin-left: 30px;
+$search-result-container-margin-right: 60px;
+$search-result-container-margin-left-on-small-screens: 10px;
+$search-result-container-margin-right-on-small-screens: 10px;
```

Comment:
On second thoughts, the name is a bit misleading since this margin applies to both the navigation and the content of the page and it should apply to _all_ pages, not just search results. `content-{left,right}-margin-{small,large}`?